staging/src/k8s.io/apiserver/pkg/server/routes/metrics.go

```diff
@@ -30,7 +31,7 @@ type DefaultMetrics struct{}
 // Install adds the DefaultMetrics handler
 func (m DefaultMetrics) Install(c *mux.PathRecorderMux) {
 	register()
-	c.Handle("/metrics", legacyregistry.Handler())
+	c.Handle("/metrics", promhttp.Handler())
```

Comment:
This seems not to bre related with the static check. This change is in #81531.
And it caused the CI failure: https://github.com/kubernetes/kubernetes/pull/102899#issuecomment-862187749

Reply:
This line gives the following check failure:
```
k8s.io/apiserver/pkg/server/routes/metrics.go:33:23: legacyregistry.Handler is deprecated: Please note the issues described in the doc comment of InstrumentHandler. You might want to consider using promhttp.Handler instead.  (SA1019)
```

I restored it to `legacyregister` and added a note.